src/main/java/util/LocalizationUtil.java

```diff
@@ -6,12 +6,13 @@
 
 public class LocalizationUtil {
 
-    static SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
-    public static final String getFormattedDate(Date date){
+    public static final String getFormattedDate(Date date,String stringDate){
```

Comment:
Passing string format to the method is not the correct way as this util tends to lose the whole point. This util should have multiple methods based on requirements. And private static final string formats as class members.

Example class  member:
Private static final String DATE_WITH_TIME = "";
Example methods:

getDatFormatWithTime(Date date)